### PR TITLE
feat: remove moderation options from research items

### PR DESCRIPTION
**Pull request by bagiyal**
PR Checklist

- [ ] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description
if someone now published research but now it became awaiting-moderation
_What this PR does_

## Git Issues

Closes #

#2571

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

File: scripts/compose-dev.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    entrypoint:
+    - sleep
+    - infinity
+    image: docker/dev-environments-default:stable-1
+    init: true
+    volumes:
+    - type: bind
+      source: /var/run/docker.sock
+      target: /var/run/docker.sock
+
```
Comment (AlfonsoGhislieri) [inline scripts/compose-dev.yaml:12]: **Issue (blocking)**: why are we adding a new script here? What is the reason behind this?

Comment (thisislawatts) [inline scripts/compose-dev.yaml:12]: **question:** This is interesting, I do not see how this is related to the change you have proposed here but I would be interested to understand how you were using this?

File: src/pages/Events/Content/EventsCreate/EventsCreate.tsx [hunk 2]
```diff
@@ -0,0 +1,329 @@
+import * as React from 'react'
+import type { RouteComponentProps } from 'react-router'
```
Comment (AlfonsoGhislieri) [inline src/pages/Events/Content/EventsCreate/EventsCreate.tsx:2]: **Issue**: I thought the issue was to do with research items why are we modifying or adding to events files?

Comment (thisislawatts) [reply src/pages/Events/Content/EventsCreate/EventsCreate.tsx:2]: Could be a merge problem, Events were all removed in: https://github.com/ONEARMY/community-platform/pull/2501

Comment (AlfonsoGhislieri) [reply src/pages/Events/Content/EventsCreate/EventsCreate.tsx:2]: Ah yes that makes sense, could you please remove this file @bagiyal

**Comment by AlfonsoGhislieri** [comment]
Some also some linting issues here @bagiyal that need to be address.

- Code is not passing lint test, you need to make sure you are using prettier: https://prettier.io/
- You need to also make sure the commits are correctly formatted, you can follow conventional commits as a guide: https://www.conventionalcommits.org/en/v1.0.0/

File: src/pages/Research/Content/Common/Research.form.tsx [hunk 3]
```diff
@@ -89,7 +89,7 @@ const ResearchForm = observer((props: IProps) => {
   }, [submissionHandler])
 
   const onSubmit = async (formValues: IResearch.FormInput) => {
-    formValues.moderation = submissionHandler.draft ? 'draft' : 'accepted' // No moderation for researches for now
+    formValues.moderation = submissionHandler.draft ? 'draft' : 'awaiting-moderation' // No moderation for researches for now --> Now Change to awaiting-moderation
     await store.uploadResearch(formValues)
```
Comment (AlfonsoGhislieri) [inline src/pages/Research/Content/Common/Research.form.tsx:93]: **Issue (blocking)**:  I don't think we should be modfying the moderation values, these should stay in their current state. What needs to be added is conditional rendering of the moderation buttons only when the status of the moderation status being set to 'awaiting-moderation' --> the logic for this has not yet been added.

Just to double check @iSCJT am I correct in these statements?

Comment (AlfonsoGhislieri) [reply src/pages/Research/Content/Common/Research.form.tsx:93]: I am on holiday so can't a zoom call I'm afraid, but here is how @iSCJT nicely summarized what you should do in the issue:

_In terms of closing this issue @bagiyal my suggestion would be to make the moderation buttons section conditional on the moderation field being equal to 'awaiting-moderation'._

So we are not looking to modify the form itself but rather just simply make the moderation buttons section render only when the field is 'awaiting-moderation'.

**Pull request closed without merging**